internal/cli/atlas/logs_download_test.go

```diff
@@ -37,14 +37,9 @@ func TestLogsDownloadOpts_Run(t *testing.T) {
 		store: mockStore,
 	}
 
-	f, err := opts.newWriteCloser()
```

Comment:
we were relying on a bug of the afero library, we don't allow to overwrite files but on our tests we call the "create" file twice, the library was updated to fix the bug and brought our tests down with it.
At the same time it makes this really hard to tests, we can't fake the call as nothing matches the `io.File` will actually pass so I had to make this test less accurate by defining will accept anything

Reply:
😢